### PR TITLE
install docs

### PR DESCRIPTION
**Pull request by sriumcp**
Signed-off-by: Srinivasan Parthasarathy <spartha@us.ibm.com>

### CODE REVIEW

File: mkdocs/docs/user-guide/topics/metrics.md [hunk 1]
```diff
@@ -10,10 +10,13 @@ Metrics in Iter8 are grouped according to the type of source (backend) from whic
...
-A Counter metric is one whose value never decreases over time. For example, the error count for an app version never decreases over the course of an experiment, and is a counter metric. 
+A Counter metric is one whose value never decreases over time. For example, the error count for an app version never decreases over the course of an experiment, and is a Counter metric. 
 
 ### Gauge
-A Gauge metric is one whose value may increase or decrease over time. For example, the error rate for an app version may increase or decrease over the course of an experiment, and is a gauge metric. 
\ No newline at end of file
+A Gauge metric is one whose value may increase or decrease over time. For example, the error rate for an app version may increase or decrease over the course of an experiment, and is a Gauge metric. 
+
+### Histogram
+A Histogram metric has a set of buckets, and counts associated with each bucket. For example, Iter8's built-in latency metric is stored in the form of a histogram. Each bucket represents a lower and upper value of observed latency value, and count represents the number of observations that fall within a bucket.
```
Comment (kalantar) [inline mkdocs/docs/user-guide/topics/metrics.md]: and counts **what** associated with each bucket?

**Review by kalantar — COMMENTED** [review]
a  comment/question
looks good otherwise.

File: mkdocs/docs/getting-started/your-first-experiment.md [hunk 2]
```diff
@@ -4,38 +4,36 @@ template: main.html
 
 # Your First Experiment
```
Comment (kalantar) [inline mkdocs/docs/getting-started/your-first-experiment.md:5]: we changed this to "quick start" in the README. Do we want the same here?

Comment (sriumcp) [reply mkdocs/docs/getting-started/your-first-experiment.md:5]: Let us keep it as is for now. I think `your second experiment`  may be desirable in the getting started section. This doesn't look like a sensible option in the GitHub README though, which is the reason for the distinction.

**Review by kalantar — APPROVED** [review]
lgtm

**Pull request merged**